WordPress/src/main/java/org/wordpress/android/datasets/ReaderBlogTable.java

```diff
@@ -382,25 +382,6 @@ public static boolean isTimeToUpdateBlogInfo(ReaderBlog blogInfo) {
         return (minutes >= ReaderConstants.READER_AUTO_UPDATE_DELAY_MINUTES);
     }
 
-    /**
```

Comment:
This was code was added in the first PR and can safely be removed because it is not used.